src/pseudopeople/data/incorrect_select_options.csv

```diff
@@ -1,11 +1,11 @@
-state,relationship_to_reference_person,sex,race_ethnicity,tax_form,event_type
```

Comment:
where does this csv actually come from?

Reply:
This is all our incorrect selection options RT provided.

Reply:
> where does this csv actually come from?

Originally the concept model linked to a [.csv in the research repo](https://github.com/ihmeuw/vivarium_research_prl/blob/main/src/vivarium_research_prl/noise/incorrect_select_options.csv) that listed all the options for "incorrect selection" / "choose the wrong option" noise. I [recently removed the reference to this .csv](https://github.com/ihmeuw/vivarium_research/pull/1287) from the concept model because it was redundant and out of date. Now the [description of "Incorrect select" noise in the concept model](https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_census_synthdata/concept_model.html#column-noise:~:text=For%20those%20selected%2C%20randomly%20select%20a%20new%20option%20from%20the%20list%20of%20all%20possible%20options.) just says,

> For those selected, randomly select a new option from the list of all possible options.

and the actual lists of options are provided in the relevant sections of the concept model. So it's up to the engineers to maintain those lists in the code however they see fit.